tests/e2e/specs/plugin-reset.test.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * WordPress dependencies
+ */
+import { deactivatePlugin, activatePlugin, visitAdminPage } from '@wordpress/e2e-test-utils';
+
+describe( 'Reset plugin', () => {
+	beforeAll( async() => {
+		await activatePlugin( 'e2e-tests-auth-plugin' );
+	} );
+
+	afterAll( async() => {
+		await deactivatePlugin( 'e2e-tests-auth-plugin' );
+	} );
+
+	beforeEach( async() => {
+		await visitAdminPage( 'admin.php', 'page=googlesitekit-settings' );
+		await expect( page ).toMatchElement( '.googlesitekit-page-header__title', { text: 'Settings' } );
+
+		await page.waitForSelector( 'button.mdc-tab' );
+
+		// Click on Admin Settings Tab.
+		await expect( page ).toClick( 'button.mdc-tab', { text: 'Admin Settings' } );
+		await page.waitForSelector( '.googlesitekit-settings-module__footer' );
+	} );
+
+	it( 'On Reset Site Kit click, confirm dialog displays.', async() => {
```

Comment:
Test names should start with a lowercase letter, makes for nice readable and consistent reports 🙂 

Think of it as a sentence: `it ...`, e.g. `it should display a confirmation dialog`, `it should hide the confirmation dialog on cancel`, `it should disconnect Site Kit`

Reply:
Updated 👍

Reply:
Excellent, thanks!